### DISCUSSION TITLE
Simulation ends immediately, empty gamslog

### DISCUSSION THREAD

**Discussion by jacobjahnkeES** [NREL/ReEDS-2.0#185] [Q&A: Execution] [answered]
Hello,
I have been trying to use ReEDS and am running into a problem where I start the basic reference case or test case or any other case, and the simulation ends almost immediately after starting. The gamslog just contains:

computer,repo,branch,commit,description
{file_path}.git,main,e65ed5ed4ffff973071839481309f77d12d802cd,v2023-19-ge65ed5e

And then the consol says: _Ref has finished
So I am not quite sure what is going on. I tried re-cloning the repo, making the env and following all the steps I could. Any ideas? Other ways to error check?

**Comment by kennedy-mindermann** [comment, marked as answer]
To perform additional error checking, I would start by running the first line that invokes a Python script in the `.bat` or `.sh` file located in the respective run folder:
/ReEDS-2.0/runs/{batch_name}\_{scenario}/call\_{batch_name}\_{scenario}.sh.

This should correspond to line 12 or so of the file, which looks something like:
```
python {path_to_run}/input_processing/copy_files.py ...
```

**Reply by jacobjahnkeES** [reply]
Thanks, I have tried going through a few of the lines in that file and they seemed to work and I was getting logs in the gamslog. But, when I try to run the `python runbatch.py` I still get the same problem.

**Reply by jacobjahnkeES** [reply]
I pulled the latest changes again and things worked. Not sure what the problem was before... but no need to worry about this for now. Thanks!
